runtime/gc_glue_java/ArrayletObjectModel.cpp

```diff
@@ -44,18 +45,26 @@ GC_ArrayletObjectModel::AssertBadElementSize()
 	Assert_MM_unreachable();
 }
 
+void
+GC_ArrayletObjectModel::AssertContiguousArrayDataUnreachable()
+{
+	Assert_MM_unreachable();
+}
+
 void
 GC_ArrayletObjectModel::AssertArrayletIsDiscontiguous(J9IndexableObject *objPtr)
 {
 #if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
 	if (!isDoubleMappingEnabled())
-#endif /* J9VM_GC_ENABLE_DOUBLE_MAP */
+#endif /* defined(J9VM_GC_ENABLE_DOUBLE_MAP) */
 	{
-		MM_GCExtensionsBase* extensions = MM_GCExtensionsBase::getExtensions(_omrVM);
-		UDATA arrayletLeafSize = _omrVM->_arrayletLeafSize;
-		UDATA remainderBytes = getDataSizeInBytes(objPtr) % arrayletLeafSize;
-		if (0 != remainderBytes) {
-			Assert_MM_true((getSpineSize(objPtr) + remainderBytes + extensions->getObjectAlignmentInBytes()) > arrayletLeafSize);
+		if (!isVirtualLargeObjectHeapEnabled()) {
+			UDATA arrayletLeafSize = _omrVM->_arrayletLeafSize;
```

Comment:
Replace UDATA to uintptr_t in ArrayletObjectModel(Base).cpp files. Corresponding *hpp files aready use it.
Also search for some other common format problems and fix them, like:
- '*' being next to type rather then next to variable
- missing space between 'if' and '('